presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloMetadata.java

```diff
@@ -409,18 +410,9 @@ private ConnectorTableMetadata getTableMetadata(SchemaTableName tableName)
 
     private List<SchemaTableName> listTables(ConnectorSession session, SchemaTablePrefix prefix)
     {
-        // List all tables if schema or table is null
-        if (prefix.getSchemaName() == null || prefix.getTableName() == null) {
-            return listTables(session, prefix.getSchemaName());
-        }
-
-        // Make sure requested table exists, returning the single table of it does
-        SchemaTableName table = new SchemaTableName(prefix.getSchemaName(), prefix.getTableName());
-        if (getTableHandle(session, table) != null) {
-            return ImmutableList.of(table);
-        }
-
-        // Else, return empty list
-        return ImmutableList.of();
+        return listTables(session, prefix.getSchemaName())
```

Comment:
This is a behavior change - the secondary check to get the table handle is omitted now. Why did you make this change?